cypress/e2e/components/header.cy.js

```diff
@@ -49,27 +44,26 @@ describe('check headerimage and content', () => {
     });
     context('check topheader images', () => {
         it('check header image', () => {
-            cy.inHeader(() => {
-                const headerImage = cy.get('img.topHeader__image');
-                headerImage
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get('img')
+
                     .should('be.visible')
                     .should('have.attr', 'src', 'img/header.jpg');
             });
         });
         it('check header logo', () => {
-            cy.inHeader(() => {
-                const logoArea = cy.get('div.topHeader__logoArea');
-                logoArea.should('be.visible');
-                logoArea.within(() => {
-                    cy.get('svg');
-                    cy.get('p').should('have.length', 2);
-                });
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get(componentSelectors.logoArea)
+                    .should('be.visible')
+                    .within(() => {
+                        cy.get('svg');
+                        cy.get('p').should('have.length', 2);
+                    });
             });
         });
         it('check header and logo sizes', () => {
-            cy.inHeader(() => {
-                const headerImage = cy.get('img.topHeader__image');
-                headerImage.invoke('width').should('be.lessThan', 1920);
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get('img').invoke('width').should('be.lessThan', 1920);
```

Comment:
hier fehlt noch ein test, ob das logo "frei" schwebt

Reply:
behoben aber bitte nochmal checken

Reply:
```
cy.get(componentSelectors.logoArea).should(
                    'have.css',
                    'position',
                    'absolute',
                );
```
guter Ansatz, aber hierzu drei Fragen:
- dieser Test klappt fürbeide breakpoints, ist das gewollt 
- teste hier doch auch bitte, ob right gesetzt ist und > 0 ist. Das ist ja der unterschied zu Desktop small
- warum ein weiteres mal nen `within`? Das geht alles in ein within rein.

Reply:
- within entfernt
- test für beide breakpoints war so nicht gedacht. es sollte geprüft werden, ob das logo mal rechts am rand ist und mal auf dem hintergrundbild. dann habe ich die element.style eigenschaften nicht verstanden
- den > 0 abgleich kriege ich nicht zum laufen

Reply:
du hast das zweite within jetzt entfernt, aber wofür ist denn jetzt Zeile 67?
Zeile 68-70 müssen jetzt natrülich in den `within` Block darüber, sonst wird ja wieder im wilden westen selektiert.
Tipp für die überprüfung des `right` attributs:
```
cy.get(componentSelectors.logoArea)
.invoke('css', 'right')
.then((str) => parseInt(str))
.should('be.greaterThan', 0);
```
das kannst du auch mit der überprüfung von position absulute chainen